build/run.js

```diff
@@ -121,7 +133,7 @@ commands.build.rust = async function(argv) {
     let crate     = argv.crate || DEFAULT_CRATE
     let crate_sfx = crate ? ` '${crate}'` : ``
     console.log(`Building WASM target${crate_sfx}.`)
-    let args = ['build','--target','web','--no-typescript','--out-dir',paths.dist.wasm.root,'--out-name','ide',crate]
```

Comment:
What was this? :O That is so strange such option was there

Reply:
Now it's gone, and we are better off.